doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst

```diff
@@ -535,24 +543,125 @@ fulfill plant nitrogen demands.
 Harvest
 ''''''''''''''''''''''''''''''
 
-Variables track the flow of grain C and
-N to food and of all other plant pools, including live stem C and N, to litter. Putting live
-stem C and N into the litter pool is in contrast to the approach for unmanaged PFTs which
-puts live stem C and N into dead stem pools first. Leaf and root C and N pools
-are routed to the litter pools in the same manner as natural vegetation. Whereas food C and N
-was formerly transferred to the litter pool, CLM5 routes food C and N
+Variables track the flow of grain C and N to food and of all other plant pools, including live stem C and N, to litter, and to biofuel feedstock.
+A fraction (determined by :math:`biofuel\_harvfrac`) of leaf/livestem C and N from bioenergy crops is removed at harvest for biofuels 
+(Equations :eq:`25.9`, :eq:`25.10`, :eq:`25.12`, and :eq:`25.13`),
+with the remaining portions go to the litter pools (Equations :eq:`20.14)`, :eq:`25.11`, and :eq:`25.14`).
+Putting live stem C and N into the litter and biofuel pools is in contrast to the approach for unmanaged PFTs which
+puts live stem C and N into dead stem pools first. 
+Leaf C and N pools are routed to the little and biofuel pools, contrast to that of unmanaged PFTs which put leaf C and N to litter pools only.
+Root C and N pools are routed to the litter pools in the same manner as natural vegetation.
+  
+.. math::
```

Comment:
For the equations below, CF, CS, NF, and NS are not previously defined in this chapter. I’m curious if these are the same/similar variables described in the previous paragraph (N retranslocation). If so, can you update the variable name for accuracy within this chapter? If not, please provide a simple definition for what these variables are, similar to the variables above in the N retranslocation section.